core/assets/bundles/bundle_fr.properties

```
@@ -1287,9 +1287,9 @@ rules.unitdamagemultiplier = Multiplicateur de Dégât des Unités
 rules.unitcrashdamagemultiplier = Multiplicateur de Dégât de chute des Unités
 rules.solarmultiplier = Multiplicateur de l'Efficacité des Panneaux Solaires
 rules.unitcapvariable = Les Noyaux contribuent à la limite d'Unités actives
-rules.unitcap = Limite d'Unités actives de Base
-rules.limitarea = Limite de la Zone de Jeu de la Carte
-rules.enemycorebuildradius = Périmètre de Non-Construction autour du Noyau ennemi :[lightgray] (blocs)
+rules.unitcap = Limite d'Unités actives de base
```

```suggestion
rules.unitcap = Limite initiale d'Unités actives
```
Maintenant que j'y pense, cette traduction n'a pas bcp de sens.
Que penses tu de celle-là, est-elle plus claire ?